Mathlib/Algebra/Category/MonCat/Basic.lean

```diff
@@ -294,6 +294,8 @@ example (R : CommMonCat.{u}) : R ⟶ R :=
 
 variable {X Y : Type u}
 
+attribute [local ext] ConcreteCategory.hom_ext
+
```

Comment:
Can this now be removed?

Reply:
I've removed it and it still works. :+1: